### PR TITLE
SEO and minor usability improvements.

### PR DESCRIPTION
**Pull request by mlkirkpatrick**
- Wrap navigation elements in nav instead of div/span.
- Display links to all pages for pagination.
- Add item name to header on item pages within a collection.
- Unique titles (browser title) for each page within a collection.
- Parse HTML for items displayed on Search page.
- Modify/add h tags to reflect page hierarchy.
- Add rel="nofollow" to print link so bots won't index it.
- Add pagination to top of item list/grid (in addition to bottom).
- Allow clearing search by "searching" with an empty string.
- Fix long names in "Filter Results" from pushing the count element down.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <meta name="google-site-verification" content="qq7fBdp8zPSI3R9_LFQ4YFmBzTw7Etg5M5HbSgZ_hak" />
     <meta name="google-site-verification" content="zISdamiilUYTQJfVqU4OSM85zEDAJTcioljTdgbaCxo" />
     <meta name="google-site-verification" content="df7oJV_ryb0hmj991vCUHHgEkZs4_PxsqKbobpHEp8U" />
+    <meta name="google-site-verification" content="4G8TYzyR2-8KWRdA3F1LRXv9l1_sBvV3YS4UZ7115Po" /> <!-- Michael K -->
```
Comment (danthewolfe) [inline public/index.html]: Remove `<!-- Michael K -->` and change meta tag to:
`<meta name="google-site-verification" content="4G8TYzyR2-8KWRdA3F1LRXv9l1_sBvV3YS4UZ7115Po" user="mk" />` 

I've made separate notes to update the other verification tags with the correct users.

File: public/index.html [hunk 2]
```diff
@@ -8,6 +8,7 @@
     <meta name="google-site-verification" content="qq7fBdp8zPSI3R9_LFQ4YFmBzTw7Etg5M5HbSgZ_hak" />
```
Comment (danthewolfe) [inline public/index.html]: `<meta name="google-site-verification" content="qq7fBdp8zPSI3R9_LFQ4YFmBzTw7Etg5M5HbSgZ_hak" user="rf" />`

File: public/index.html [hunk 3]
```diff
@@ -8,6 +8,7 @@
...
     <meta name="google-site-verification" content="zISdamiilUYTQJfVqU4OSM85zEDAJTcioljTdgbaCxo" />
```
Comment (danthewolfe) [inline public/index.html]: `<meta name="google-site-verification" content="zISdamiilUYTQJfVqU4OSM85zEDAJTcioljTdgbaCxo" user="jh" />`

File: public/index.html [hunk 4]
```diff
@@ -8,6 +8,7 @@
...
     <meta name="google-site-verification" content="df7oJV_ryb0hmj991vCUHHgEkZs4_PxsqKbobpHEp8U" />
```
Comment (danthewolfe) [inline public/index.html]: `<meta name="google-site-verification" content="df7oJV_ryb0hmj991vCUHHgEkZs4_PxsqKbobpHEp8U" user="rd"/>`

File: src/components/Search/ListItem.jsx [hunk 5]
```diff
@@ -21,16 +22,16 @@ const MyListItem = createReactClass({
...
+        { RemoveMarkup(this.props.item.name) }
+      </h3>
     )
   },
 
   secondaryText: function () {
     return (
       <span style={{ maxWidth: '50em', marginLeft: '30px' }}>
-        { this.props.item.description }
+        { RemoveMarkup(this.props.item.description) }
```
Comment (danthewolfe) [inline src/components/Search/ListItem.jsx]: Let's cleanup the spacing here on this list view. It's probably a byproduct of `RemoveMarkup`:
screen shot 2018-10-18 at 10 50 23 am

File: public/index.html (hunk 1 above)
Comment (danthewolfe) [reply public/index.html]: I think I'm manually verified, but please also add:
`<meta name="google-site-verification" content="kiUlnvDP0sZ96LCd5lRAbBvMxYVje53ppg2omqweX2Q" user='dw'/>`

File: src/components/Search/ListItem.jsx (hunk 5 above)
Comment (mlkirkpatrick) [reply src/components/Search/ListItem.jsx]: Actually just a little css issue. Fixed in https://github.com/ndlib/beehive/pull/298/commits/95318b684586d509953dc87d893a644aac48843d.

File: src/components/Search/ListItem.jsx [hunk 6]
```diff
@@ -21,16 +22,16 @@ const MyListItem = createReactClass({
 
   primaryText: function () {
     return (
-      <span style={{ marginLeft: '30px', display: 'inline-block' }}>
-        {this.props.item.name}
-      </span>
+      <h3 style={{ marginLeft: '30px', display: 'inline-block' }}>
+        { RemoveMarkup(this.props.item.name) }
```
[the remark this answers is not in the document]
Comment (mlkirkpatrick) [reply src/components/Search/ListItem.jsx]: HTML markup is showing up in the description field. (Namely, <p> tags and the occasional &nbsp;) Probably not needed on the name field, now that you mention it.

**Pull request merged**